ehr/resources/web/ehr/model/sources/ParentChild.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2018 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+EHR.model.DataModelManager.registerMetadata('ParentChild', {
```

Comment:
A comment here to describe the intended use case would be helpful context.

Reply:
Added